basic_pitch/constants.py

```diff
@@ -59,3 +61,5 @@ def _freq_bins(bins_per_semitone: int, base_frequency: float, n_semitones: int)
 
 FREQ_BINS_NOTES = _freq_bins(NOTES_BINS_PER_SEMITONE, ANNOTATIONS_BASE_FREQUENCY, ANNOTATIONS_N_SEMITONES)
 FREQ_BINS_CONTOURS = _freq_bins(CONTOURS_BINS_PER_SEMITONE, ANNOTATIONS_BASE_FREQUENCY, ANNOTATIONS_N_SEMITONES)
+
+Split = Enum("Split", ["train", "validation", "test"])
```

Comment:
```suggestion
class Split(Enum):
    train = "train"
    validation = "validation"
    test = "test"
```

Usually I prefer an enum class cause the editor's language server can better understand the code

Reply:
you mean vscode?

Reply:
Any editor with lsp